python/test/t/geometry/test_trianglemesh.py

```diff
@@ -35,3 +37,46 @@ def test_clip_plane():
     clipped_cube = cube.clip_plane(point=[0.5, 0, 0], normal=[1, 0, 0])
     assert clipped_cube.vertex['positions'].shape == (12, 3)
     assert clipped_cube.triangle['indices'].shape == (14, 3)
+
+
+def test_create_box():
+    # test with default parameters
+    box_default = o3d.t.geometry.TriangleMesh.create_box()
+
+    vertex_positions_default = o3c.Tensor(np.array([[0.0, 0.0, 0.0],
+                                                    [1.0, 0.0, 0.0],
+                                                    [0.0, 0.0, 1.0],
+                                                    [1.0, 0.0, 1.0],
+                                                    [0.0, 1.0, 0.0],
+                                                    [1.0, 1.0, 0.0],
+                                                    [0.0, 1.0, 1.0],
+                                                    [1.0, 1.0, 1.0]]),
+                                          dtype=o3c.Dtype.Float64,
+                                          device=o3c.Device("CPU:0"))
+
+    triangle_indices = o3c.Tensor(np.array([[4, 7, 5], [4, 6, 7], [0, 2, 4],
+                                            [2, 6, 4], [0, 1, 2], [1, 3, 2],
+                                            [1, 5, 7], [1, 7, 3], [2, 3, 7],
+                                            [2, 7, 6], [0, 4, 1], [1, 4, 5]]),
+                                  dtype=o3c.Dtype.Int64,
+                                  device=o3c.Device("CPU:0"))
+
+    assert box_default.vertex['positions'].allclose(vertex_positions_default)
+    assert box_default.triangle['indices'].allclose(triangle_indices)
+
+    # test with custom parameters
+    box_custom = o3d.t.geometry.TriangleMesh.create_box(2, 4, 3)
+
+    vertex_positions_custom = o3c.Tensor(np.array([[0.0, 0.0, 0.0],
```

Comment:
you may remove `np.array`. A tensor can be directly initialized using a list. Same for the rest of PR.